middleware/kubernetes/README.md

```diff
@@ -115,6 +115,12 @@ kubernetes coredns.local {
 	# a path to a file structured like resolv.conf.
 	upstream 12.34.56.78:53
 	
+	# federation <federation-name> <federation-domain>
```

Comment:
Regarding visual formatting: I can move in-line comments into separate sections.
Can you point to an example README format that I should follow.

Regarding expansion: What areas need expansion/improvement? Federation I presume, and others...